source/PlayerInfo.cpp

```diff
@@ -4149,11 +4154,12 @@ void PlayerInfo::SortAvailable()
 					return false;
 			}
 			// Tiebreaker for equal PAY is ABC.
+			// This uses the Mission class' overridden comparison operator, which orders by 'order' field then name.
 			case ABC:
 			{
-				if(lhs.Name() < rhs.Name())
+				if(&lhs < &rhs)
 					return true;
-				else if(lhs.Name() > rhs.Name())
+				else if(&lhs > &rhs)
```

Comment:
This is only for the jobs board, which I think should remain purely alphabetical. The whole purpose of the `order` keyword is to manipulate the order in which non-job mission offer, not manipulate the order in which job missions appear on the jobs board.